src/lib/simulation.js

```diff
@@ -33,6 +33,24 @@ function patchAugmintBalances(balances) {
 }
 
 function init(initParams) {
+    // console.log("before clear");
+    // augmint.debugState();
+    // showLog();
+
+    iteration = 0;
+    clock.setTime(0);
+
+    for (var it = actors.values(), val = null; (val = it.next().value); ) {
+        if (val.locks.length > 0) {
+            console.log(val.id + ":" + JSON.stringify(val.locks[0].id));
+            val.releaseACD(val.locks[0].id);
```

Comment:
what are you trying to achieve here? why not just remove all actors?

Reply:
already removed in the last version: 
https://github.com/Augmint/augmint-sim/blob/feature_restart/src/lib/simulation.js